### PR TITLE
fix(docs): updates for contributing to recipes and docs

### PR DESCRIPTION
**Pull request by marcysutton**
## Description

We've gotten some repeated questions about recipes and I've seen a few PRs linking broken images to GitHub, so I updated the contributing docs to clarify these items.

## Related Issues

N/A

### CODE REVIEW

File: docs/contributing/docs-contributions.md [hunk 1]
```diff
@@ -28,6 +29,14 @@ When working on the Gatsby.js documentation, you can choose between two major st
...
 - Clone the Gatsby.js repo and get the `www` site up and running locally. This is necessary for more thorough documentation content and infrastructure changes. Learn how to get set up in the [Gatsby docs setup instructions](#docs-site-setup-instructions).
 
+## Fixing image and link paths
+
+If you find a broken image URL in the Gatsby docs, it should be fixed and kept relative to the site source rather than linked to the remote repo on GitHub. This ensures that when the site is deployed, all images are included in the build.
+
+To address missing images, check the doc or tutorial source [in the Gatsby repo](https://github.com/gatsbyjs/gatsby/docs) to see if it was moved in its history and if the images are still in its old location. Check to see if those images are also referenced from more than once doc. If they aren't, move them to the new directory location (and update URL references relative to the content, if necessary). If they are referenced in more than one location, use relative paths and don't duplicate images.
```
Comment (gillkyle) [inline docs/contributing/docs-contributions.md]:
```suggestion
To address missing images, check the doc or tutorial source [in the Gatsby repo](https://github.com/gatsbyjs/gatsby/docs) to see if it was moved in its history and if the images are still in its old location. Check to see if those images are also referenced from more than one doc. If they aren't, move them to the new directory location (and update URL references relative to the content, if necessary). If they are referenced in more than one location, use relative paths and don't duplicate images.
```

File: docs/contributing/docs-templates.md [hunk 2]
```diff
@@ -195,18 +195,20 @@ Assume the reader has basic programming knowledge like the command line, code ed
...
 The components of a recipe are:
 
 - The name of the recipe, which should describe a single task
-- Requirements and prerequisites
+- A list of requirements and prerequisites
 - Step-by-step directions
 - A link to a working example
 
-Recipes should be short. If you're finding a recipe is becoming too long to fit on the Docs Recipes page due to including many prerequisites or steps, consider writing a tutorial instead.
+Recipes should be short. This is accomplished by limiting steps to what is unique to the task at-hand; prerequisites and requirements should be mentioned but not include install steps for things like npm or Gatsby CLI. Linking to full reference guide, tutorial, or or working example can complete the loop for anyone who needs more help.
```
Comment (gillkyle) [inline docs/contributing/docs-templates.md]:
```suggestion
Recipes should be short. This is accomplished by limiting steps to what is unique to the task at-hand; prerequisites and requirements should be mentioned but not include install steps for things like npm or Gatsby CLI. Linking to full reference guide, tutorial, or a working example can complete the loop for anyone who needs more help.
```

**Comment by marcysutton** [comment]
Oops! My rush to get this out of the door left a few typos. They have been addressed via suggestions. Thanks @gillkyle!

**Comment by marcysutton** [comment]
Going to merge this so we can reference it in issues and PRs!

**Pull request merged**